src/App.php

```diff
@@ -163,11 +162,18 @@ protected function configureRoutes( \Slim\Slim $slim ) {
 					echo json_encode( array( 'error' => 'Unauthorized' ) );
 					$slim->stop();
 				}
+			},
+			'trailing-slash' => function () use ( $slim ) {
+				// Remove trailing slashes
+				if ( substr( $_SERVER['REQUEST_URI'], -1 ) === '/' ) {
+					$uri = preg_replace( '/\/+$/', '', $_SERVER['REQUEST_URI'] );
+					$slim->redirect( $uri );
+				}
 			}
 		);
-		$slim->group( '/', $middleware['inject-user'], $middleware['set-environment'],
+		$slim->group( '/', $middleware['trailing-slash'], $middleware['inject-user'], $middleware['set-environment'],
 			function () use ( $slim ) {
-				$slim->get( '/?', function () use ( $slim ) {
+				$slim->get( '/', function () use ( $slim ) {
```

Comment:
Shouldn't this route be for `''` now that you are ensuring that there are no trailing slashes? Since you are in the `'/'` group this seems like it should only match an URL of `/tool-name//`.